tests/snapshot_tests/test_snapshots.py

```diff
@@ -1450,8 +1450,23 @@ def test_split(snap_compare):
     assert snap_compare(SNAPSHOT_APPS_DIR / "split.py", terminal_size=(100, 30))
 
 
+def test_system_commands(snap_compare):
+    """Test help panel."""
```

Comment:
Remember that these docstrings appear in the snapshot report now, so it's useful to describe what we expect to see in the snapshot. If it fails, it can sometimes be hard to know what we expect the snapshot to look like.

```suggestion
    """The system commands should appear in the command palette."""
```